doc/source/docs/user_guide/fiona_to_pyogrio.rst

```diff
@@ -0,0 +1,69 @@
+.. currentmodule:: geopandas
+
+.. ipython:: python
+   :suppress:
+
+   import geopandas
+
+Migration from the Fiona to the Pyogrio IO engine
+=================================================
+
+Since version 0.11, GeoPandas supports two IO engines to read and write files:
+`Fiona <https://fiona.readthedocs.io>`__ and
+`Pyogrio <https://pyogrio.readthedocs.io>`__.
+
+For Geopandas versions <1.0, GeoPandas defaults to use Fiona. It is possible to use
+Pyogrio using the ``engine="pyogrio"`` parameter in :func:`geopandas.read_file` and
+:func:`geopandas.GeoDataFrame.to_file`. You can also change the default engine globally
+with:
+
+.. code-block:: python
+
+    geopandas.options.io_engine = "pyogrio"
+
+Starting from GeoPandas 1.0, the global default will change from Fiona to Pyogrio. The
```

Comment:
Make present tense. This will come together with 1.0 so "will" won't make sense at that point.